src/app/home/leadingInnovators/index.scss

```diff
@@ -0,0 +1,41 @@
+.gradient-text-leading{
+    line-height: 1.3;
+    background: linear-gradient(266deg, #FAE2B6 5.93%, #EF90B8 44.82%, #B684F0 68.16%);
+}
+
+.icon-award{
+    padding-top: 30px;
+    display: flex !important;
+    justify-content: center !important;
+}
+
+.animation-card{
+    transition: .2s;
+    animation-duration: 1s;
+    animation-name: identifier;
+
+    @keyframes identifier {
```

Comment:
There is no need to use animation and keyframes. Using width and transition is sufficient